.github/workflows/dev-deploy.yaml

```
@@ -22,11 +22,14 @@ env:
     PROJECT_ID: ${{ secrets.PROJECT_ID }}
     ENCRYPTION_SECRET: ${{ secrets.ENCRYPTION_SECRET }}
     STRAVA_SECRET: ${{ secrets.STRAVA_SECRET }}
+    INFRASTRUCTURE_SECRET: ${{ secrets.INFRASTRUCTURE_SECRET }}
+    SHOULD_SETUP: ${{ steps.check_changes.outputs.changes_detected == 'true' || (github.event_name == 'workflow_dispatch' && github.event.inputs.force_deploy
```

actionlint (1.7.4)

context "steps" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)